lib/gtk/v3_4/gtk_core.nit

```diff
@@ -129,539 +133,578 @@ extern class GtkWidget `{GtkWidget *`}
-		return gtk_window_get_resizable( recv );
+	fun resizable: Bool `{
+		return gtk_window_get_resizable(recv);
 	`}
 
-	fun resizable=( is_resizable : Bool) is extern `{
-		return gtk_window_set_resizable( recv, is_resizable );
+	fun resizable=(is_resizable: Bool) `{
+		return gtk_window_set_resizable(recv, is_resizable);
 	`}
 
-	#Activates the current focused widget within the window.
-	#returns TRUE if a widget got activated.
-	fun activate_focus : Bool is extern `{
-		return gtk_window_activate_focus( recv );
+	# Activates the current focused widget within the window.
+	# returns TRUE if a widget got activated.
+	fun activate_focus: Bool `{
+		return gtk_window_activate_focus(recv);
 	`}
 
-	#Sets a window modal or non-modal. Modal windows prevent interaction with other windows in the same application.
-	fun modal=( is_modal : Bool ) is extern `{
-		gtk_window_set_modal( recv, is_modal );
+	# Sets a window modal or non-modal. Modal windows prevent interaction with other windows in the same application.
+	fun modal=(is_modal: Bool) `{
+		gtk_window_set_modal(recv, is_modal);
 	`}
 
-	#Windows can't actually be 0x0 in size, they must be at least 1x1
-	#but passing 0 for width and height is OK, resulting in a 1x1 default size.
-	#params width in pixels, or -1 to unset the default width
-	#params height in pixels, or -1 to unset the default height
-	fun default_size( width : Int, height : Int ) is extern `{
-		gtk_window_set_default_size( recv, width, height );
+	# Windows can't actually be 0x0 in size, they must be at least 1x1
+	# but passing 0 for width and height is OK, resulting in a 1x1 default size.
+	# params width in pixels, or -1 to unset the default width
+	# params height in pixels, or -1 to unset the default height
+	fun default_size(width: Int, height: Int) `{
+		gtk_window_set_default_size(recv, width, height);
 	`}
 
-	#Activates the default widget for the window
-	#unless the current focused widget has been configured to receive the default action (see gtk_widget_set_receives_default()), in which case the focused widget is activated.
-	fun activate_default : Bool is extern `{
-		return gtk_window_activate_default( recv );
+	# Activates the default widget for the window
+	# unless the current focused widget has been configured to receive the default action (see gtk_widget_set_receives_default()), in which case the focused widget is activated.
+	fun activate_default: Bool `{
+		return gtk_window_activate_default(recv);
 	`}
 
-	fun gravity : GdkGravity is extern `{
-		return gtk_window_get_gravity( recv );
+	fun gravity: GdkGravity `{
+		return gtk_window_get_gravity(recv);
 	`}
 
-	fun gravity=( window_grav : GdkGravity ) is extern `{
-		gtk_window_set_gravity( recv, window_grav );
+	fun gravity=(window_grav: GdkGravity) `{
+		gtk_window_set_gravity(recv, window_grav);
 	`}
 
-#	fun position : GtkWindowPosition is extern `{
-#		return gtk_window_get_position( recv );
+#	fun position: GtkWindowPosition `{
```

Comment:
They are not available on GTK 3.4 or 3.6. They need to be moved to a different version.